userguide/kaui/Kaui_Guide_Draft.adoc

```diff
@@ -1070,13 +1128,19 @@ Kaui generates and displays the draft invoice. A message at the top indicates it
 
 === Payments
 
-*_IN PROCESS_*
+This section helps you become familiar with the Payments and Payment Details pages.
+
+==== Payments Page
+The Payments page lists any payments applied to invoices for the customer account.
+

```

Comment:
You've got a very good point. I blew past these screens, in a hurry to describe the editable ones! Let me ask P and S about it, as it will be more work, and they would like me to finish it soon.